README.md

```diff
@@ -5,47 +5,64 @@
 <p align="center">
     <img alt="PyPI" src="https://img.shields.io/pypi/v/chatgpt-api">
     <img alt="License" src="https://img.shields.io/github/license/mbroton/chatgpt-api">
-    <img alt="Coverage" src="https://img.shields.io/badge/coverage-100%25-green">
+    <img alt="Coverage" src="https://img.shields.io/badge/coverage-85%25-green">
```

Comment:
Looks like you didn't have latest changes on your branch, because a few changes in this file are in reverse to what I've committed recently (line 8, 16, 23, 29 and 118)

Reply:
This should be ok now.